packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -416,19 +429,36 @@ export default class PciInstancesAddController {
     return this.$q.when();
   }
 
-  loadPrivateNetwork() {
-    if (this.model.datacenter.type === this.LOCAL_ZONE_REGION) {
-      this.PciProjectsProjectInstanceService.getLocalPrivateNetworks(
-        this.projectId,
-        this.model.datacenter.name,
+  getLocalPrivateNetworkSubnet() {
+    return this.$q
+      .all(
+        this.availableLocalPrivateNetworks.map((privateNetwork) => {
+          if (privateNetwork.id) {
+            this.PciProjectsProjectInstanceService.getLocalPrivateNetworkSubnets(
```

Comment:
Don't you have to return the result of this Promise as well ?
```suggestion
            return this.PciProjectsProjectInstanceService.getLocalPrivateNetworkSubnets(
```

Reply:
yup Thanks